joblib/memory.py

```diff
@@ -593,19 +593,23 @@ def __getstate__(self):
         return state
 
     def check_call_in_cache(self, *args, **kwargs):
-        """Check if function call is in the memory cache.
+        """Check if the function call is cached and valid for given arguments.
+
+        Does not call the function or do any work besides function inspection
+        and argument hashing.
 
-        Does not call the function or do any work besides func inspection
-        and arg hashing.
+        - Compare the function code with the one from the cached function,
+        asserting if it has changed.
```

Comment:
I think the doc build failure is coming from this line

```suggestion
          asserting if it has changed.
```